examples/electron/package.json

```diff
@@ -60,7 +60,7 @@
     "watch": "concurrently -n compile,bundle \"theiaext watch --preserveWatchOutput\" \"theia build --watch --mode development\"",
     "start": "theia start --plugins=local-dir:../../plugins",
     "start:debug": "yarn start --log-level=debug",
-    "test": "electron-mocha --timeout 60000 \"./lib/test/**/*.espec.js\""
+    "test": "electron-mocha --no-sandbox --timeout 60000 \"./lib/test/**/*.espec.js\""
```

Comment:
Looking quickly I do not think this is how it's done for electron-mocha. Rather it looks like you need to provide a `.js` file that defined the required argument(s).

https://github.com/jprichardson/electron-mocha#webgl-tests

Reply:
I tested before in gitpod (it failed without the flag), and with it it passed successfully: https://github.com/eclipse-theia/theia/pull/8415#issuecomment-685853207. I'll see if it passes the build, and if not I'll take a look at what you proposed.